tools/accuracy_checker/configs/bert-small-uncased-whole-word-masking-squad-0001.yml

```diff
@@ -0,0 +1,45 @@
+models:
+  - name: bert-small-uncased-whole-word-masking-squad-0001
+    launchers:
+      - framework: dlsdk
+        adapter:
+          type: bert_question_answering
+          start_token_logits_output: "output_s"
+          end_token_logits_output: "output_e"
+        inputs:
+          - name: "input_ids"
+            type: INPUT
+            value: "input_ids"
+          - name: "attention_mask"
+            type: INPUT
+            value: 'input_mask'
+          - name: 'token_type_ids'
+            type: INPUT
+            value: 'segment_ids'
+
+    datasets:
+      - name: squad
+        data_source: squad_v1.1
+        annotation_conversion:
+          converter: squad
+          testing_file: squad_v1.1/dev-v1.1.json
+          vocab_file: bert-large-uncased-whole-word-masking-squad-onnx/vocab.txt
+          max_seq_length: 384
+          max_query_length: 64
+          doc_stride: 128
+          lower_case: true
```

Comment:
this section should be moved to dataset_definitioons, here only dataset name should be stayed

Reply:
looks strange to split dataset description into 2 files. but I did it as you requested.